src/pages/home/report/ListHeaderComponentLoader/ListHeaderComponentLoader.android.js

```diff
@@ -0,0 +1,16 @@
+import {View, ActivityIndicator} from 'react-native';
+import styles, {stylesGenerator} from '../../../../styles/styles';
+import themeColors from '../../../../styles/themes/default';
+
+function ListHeaderComponentLoader() {
```

Comment:
Why are we using a different component for loading more reportActions in the ReportActionsList header v.s: the footer. Can we instead just:

1. Create a component like `ReportActionsListLoader`, that's basically just renders `<ReportActionsSkeletonView />` like we do in the existing `ListFooterComponent`
1. Reuse the same thing for the header and the footer.

Reply:
We use an activity indicator instead of a skeleton for the header to reduce the jumping size when this indicator fades out.
Also, to avoid rerendering, I put it as an absolute view. Therefore, there are different styles for Android and the rest of the platforms, so I made a separate file.
I can create a component where would be rendered these two different options.

Reply:
Ok, yes let's create a shared component and include comments explaining the difference and why